ncats_webd/cybex_queries.py

```diff
@@ -30,6 +30,18 @@
 }
 
 
+def get_hostnames_of_tickets(db, tickets):
+    """Given a list of tickets, return a dict mapping ticket IP to hostname (if any) from latest host scan."""
+    ip_to_hostname = dict()
+    ticket_ip_ints = [int(ticket["ip"]) for ticket in tickets]
+    for host_scan in db.HostScanDoc.find({
```

Comment:
Since we only care about one field here would it make sense to use a projection?

Reply:
Good thinking (though we care about two fields, not one).  I added the projection in https://github.com/cisagov/ncats-webd/pull/34/commits/e3e3995151bb1d5d929bf6dd37ed5124d581615c.